src/traces/parcoords/calc.js

```diff
@@ -0,0 +1,49 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var hasColorscale = require('../../components/colorscale/has_colorscale');
+var calcColorscale = require('../../components/colorscale/calc');
+var Lib = require('../../lib');
+
+
+module.exports = function calc(gd, trace) {
+    var inputDimensions = trace.dimensions,
+        dimensions = [];
+    for(var i = 0; i < inputDimensions.length; i++) {
```

Comment:
I don't see anything in this file that would constrain things to element index 0 - perhaps it's meant to be linked to [this other source file](https://github.com/monfera/plotly.js/blob/133f3abc020636a0edcc08f889bb8331b6f3fc5f/src/traces/parcoords/plot.js#L22)? Several of the PR comments seem to be unrelated to the quoted code so I might get some wrong.

The latter `[0]` indexing is patterned after the `pie` [approach](https://github.com/plotly/plotly.js/blob/master/src/traces/pie/plot.js#L38) and that's where I found the data, though again, I'd be happy to get guidance on how to do it differently as it's been this way since my initial versions. As I'm unsure what source/line the comment relates to, I might be completely off though.